frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -481,6 +481,10 @@ function ReaderBookmark:onShowBookmark(match_table)
     local multilines_show_more_text = G_reader_settings:isTrue("bookmarks_items_multilines_show_more_text")
     local show_separator = G_reader_settings:isTrue("bookmarks_items_show_separator")
 
+    self.bookmark_menu = CenterContainer:new{
+        dimen = Screen:getSize(),
+        covers_fullscreen = true, -- hint for UIManager:_repaint()
```

Comment:
Btw, we may not even need this CenterContainer: it is full screen, but so is local bm_menu = Menu:new{:
width = Screen:getWidth(),
height = Screen:getHeight(),